_config.yml

```diff
@@ -81,6 +83,14 @@ authors:
     email:          kushagraagra008@gmail.com
     twitter:        https://www.instagram.com/kushagra_agra008/
 
+  kushagra:
+    name:           Kushagra Agarwal
+    site:           https://codewithkushagra.netlify.app/
+    avatar:         "assets/images/authors/kushagra.jpg"
+    bio:            "I am a code geek, a NARUTO fan and also love expressing my depression in words :)"
+    email:          kushagraagra008@gmail.com
+    twitter:        https://www.instagram.com/kushagra_agra008/
+
```

Comment:
Why have you added your info again?

Reply:
Oh, okay I didn't realised I will make the correction